experiments/spotify/3_devlop_own_source/README.md

```diff
@@ -0,0 +1,80 @@
+# Developing a pipeline with a custom (own) source
+This is example of a pipeline that includes custom source. The pipeline will be developed interatively by letting the user to modify schemas and run it again to apply changes.
+At the end we'll consume a few custom sources to show how pipeline with many sources could work.
+
+## Pipeline init
+
+Please read READMEs for `no_code_consume_source` and `consume_source` on options to initialize pipeline. In this case we do
+```
+dlt init custom-source spotify_pipeline
```

Comment:
We need to align on names and better delineate on custom sources for yourself vs. custom sources for others. I have been calling the first 'snippets' (though I kind of like 'scripts' that use the DLT library better) and the second 'sources'